### ISSUE TITLE
Create a link button with tooltip component

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/thunderbird-accounts#192]
A very simple component, can be put together with services-ui components.

Create a component that wraps an image of choice in a link. Additionally on hover a tooltip can display text. 

For screenreaders they should be covered with invisible labels or aria-label.

**Comment by MelissaAutumn** [comment]
I believe this is done.
